### PR TITLE
nextclade: add alignment example

### PR DESCRIPTION
**Pull request by corneliusroemer**
I'm a maintainer of Nextclade. We released a new major version with new capabilities.

This PR adds an example for the most important new capability.

- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** v3.1.0

### CODE REVIEW

File: pages/common/nextclade.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+> More information: <https://docs.nextstrain.org/projects/nextclade/en/stable/user/nextclade-cli/index.html>.
+
+- Align sequences to user provided [r]eference, [o]utputting the alignment to a file:
+
+`nextclade run {{path/to/sequences.fa}} -r {{path/to/reference.fa}} -o {{path/to/alignment.fa}}`
 
 - Create a TSV report, auto-downloading the latest [d]ataset:
 
-`nextclade run -d {{dataset_name}} {{path/to/fasta}} -t {{path/to/output_tsv}}`
+`nextclade run {{path/to/fasta}} -d {{dataset_name}} -t {{path/to/output_tsv}}`
```
Comment (vitorhcl) [inline pages/common/nextclade.md:12]:
```suggestion
`nextclade run {{path/to/fasta}} -d {{dataset_name}} -t {{path/to/report.tsv}}`
```

**Review by vitorhcl — APPROVED** [review]
Thanks for your contributuion! I have a minor suggestion

File: pages/common/nextclade.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 
 > Bioinformatics tool for virus genome alignment, clade assignment and qc checks.
-> More information: <https://docs.nextstrain.org/projects/nextclade/en/stable/user/nextclade-cli.html>.
+> More information: <https://docs.nextstrain.org/projects/nextclade/en/stable/user/nextclade-cli/index.html>.
+
+- Align sequences to user provided [r]eference, [o]utputting the alignment to a file:
+
+`nextclade run {{path/to/sequences.fa}} -r {{path/to/reference.fa}} -o {{path/to/alignment.fa}}`
 
 - Create a TSV report, auto-downloading the latest [d]ataset:
```
Comment (vitorhcl) [inline pages/common/nextclade.md:10]:
```suggestion
- Create a [t]SV report, auto-downloading the latest [d]ataset:
```

File: pages/common/nextclade.md (hunk 1 above)
Comment (corneliusroemer) [reply pages/common/nextclade.md:12]: thanks, great idea!

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution.

LGTM, after the above suggestions.

**Comment by kbdharun** [comment]
Hi @corneliusroemer, any updates on this?

**Comment by corneliusroemer** [comment]
Apologies for the delay, I've applied the suggestions.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**